app/Http/Requests/StoreAirlineRequest.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class StoreAirlineRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     */
+    public function authorize(): bool
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array<string, \Illuminate\Contracts\Validation\ValidationRule|array<mixed>|string>
+     */
+    public function rules(): array
+    {
+        return [
+            'name' => ['required', 'string', 'max:255', 'unique:airlines,name'],
```

Comment:
```suggestion
            'name' => ['required', 'string', 'max:255', 'unique:airlines'],
```
Given that the field matches the column, you can't skip it. It makes more sense to explicitly tell it what column it should look for when they don't match.